### PR TITLE
docs: Move and update Stake Requirements

### PR DESCRIPTION
**Pull request by amela**
- [x] Move Stake Requirements to run-a-node/prerequisites/
- [x] Add info about the Testnet Faucet
- [x] ~~Delete contribute-to-the-network/run-validator.md~~
- [x] Check links

Preview: https://deploy-preview-115--trusting-archimedes-14c863.netlify.app/general/run-a-node/prerequisites/stake-requirements

### CODE REVIEW

**Comment by matevz** [comment]
This seems like a duplicate of https://docs.oasis.dev/general/manage-tokens/advanced/oasis-cli-tools/common-staking-info? I suggest we mention it.. left a review.

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+staking threshold].
+* 100 ROSE staked for your validator node's registration since that is the 
+[current validator node's staking threshold].
+* If you want to be part of the active validator set, you will need enough 
+ROSE staked --  this means you will have to be one of the top 120 entities 
+by stake. You can check out current top 120 entities on the blockchain explorer 
+such as [Oasis Scan].
+
+
+[current entity's staking threshold]: ../../oasis-network/genesis-doc.md#node-and-paratime-token-thresholds
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md]:
```suggestion
[staking thresholds]: ../../oasis-network/genesis-doc.md#staking-thresholds
```

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+[Delegation Policy]: ../../foundation/delegation-policy.md
+
+:::
+
+Currently, you are going to need:
+
+* 100 ROSE staked for your entity's registration since that is the [current entity's 
+staking threshold].
+* 100 ROSE staked for your validator node's registration since that is the 
+[current validator node's staking threshold].
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md]:
```suggestion
[current validator node's staking threshold][staking thresholds].
```

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Foundation, see the [Delegation Policy] document.
+
+[Delegation Policy]: ../../foundation/delegation-policy.md
+
+:::
+
+Currently, you are going to need:
+
+* 100 ROSE staked for your entity's registration since that is the [current entity's 
+staking threshold].
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md]:
```suggestion
staking threshold][staking thresholds].
```

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* 100 ROSE staked for your validator node's registration since that is the 
+[current validator node's staking threshold].
+* If you want to be part of the active validator set, you will need enough 
+ROSE staked --  this means you will have to be one of the top 120 entities 
+by stake. You can check out current top 120 entities on the blockchain explorer 
+such as [Oasis Scan].
+
+
+[current entity's staking threshold]: ../../oasis-network/genesis-doc.md#node-and-paratime-token-thresholds
+[current validator node's staking threshold]: ../../oasis-network/genesis-doc.md#node-and-paratime-token-thresholds
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md]: Unify with the link above.

```suggestion
```

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+* 100 ROSE staked for your entity's registration since that is the [current entity's 
+staking threshold].
+* 100 ROSE staked for your validator node's registration since that is the 
+[current validator node's staking threshold].
+* If you want to be part of the active validator set, you will need enough 
+ROSE staked --  this means you will have to be one of the top 120 entities 
+by stake. You can check out current top 120 entities on the blockchain explorer 
+such as [Oasis Scan].
+
+
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md:31]:
```suggestion
Staking thresholds may change in the future. Read [Common staking info] chapter,
to learn how to check the current network thresholds directly from the `oasis-node`.

[Common staking info]: ../../manage-tokens/advanced/oasis-cli-tools/common-staking-info.md

```

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+The size of the consensus committee (i.e. the validator set) is configured by 
+the [**max_validators** consensus parameter].
+
+[**max_validators** consensus parameter]: ../../oasis-network/genesis-doc.md#consensus
+
+:::
+
+## Testnet
+
+For the Testnet you are going to need TEST tokens. To receive TEST tokens please
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md]:
```suggestion
For the Testnet you are going to need TEST tokens. You can receive a limited number of
TEST by using our [Oasis Network Testnet Faucet]. For more tokens please contact us at
our official [Discord #testnet channel][discord-testnet].

[discord-testnet]: https://discord.com/channels/748635004384313474/960599828662976522
```

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+the [**max_validators** consensus parameter].
+
+[**max_validators** consensus parameter]: ../../oasis-network/genesis-doc.md#consensus
+
+:::
+
+## Testnet
+
+For the Testnet you are going to need TEST tokens. To receive TEST tokens please 
+use the [Oasis Network Testnet Faucet].
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md]:
```suggestion
```

**Review by matevz — CHANGES_REQUESTED** [review]

**Comment by amela** [comment]
Yes it is similar to the https://docs.oasis.dev/general/manage-tokens/advanced/oasis-cli-tools/common-staking-info. However `Stake Requirements` is meant to be read before setting up a node and to get the feel of how many tokens someone needs to be a validator.

File: docs/general/run-a-node/prerequisites/stake-requirements.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+more tokens please contact us at our official [Discord #testnet channel]
+[discord-testnet].
+
+[faucet-testnet]: https://faucet.testnet.oasis.dev/
+[discord-testnet]: https://discord.com/channels/748635004384313474/960599828662976522
+
+
+:::info
+
+For more information about obtaining information on your entity's account, see
```
Comment (matevz) [inline docs/general/run-a-node/prerequisites/stake-requirements.md]: I would remove the admonition and move this as a normal paragraph just before line 25 (`Staking thresholds may change in the future...`).

**Review by matevz — APPROVED** [review]
Just one detail. Otherwise looks good. :+1:

**Pull request merged**